source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md

```diff
@@ -0,0 +1,67 @@
+---
+title: vagrant-service-manager 1.3.0 Released - Focus on Kubernetes Support
+author: budhram
+date: 2016-08-16 12:54:00 UTC
+tags: vagrant, devtools, releases, kubernetes
+published: true
+comments: true
+---
+
+This release of [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) introduces support for displaying Kubernetes configuration information. This enable users to access the Kubernetes server running inside ADB virtual machine from their host.
+
+Client binary installation support for 'Kubernetes' is included in this release. Support for client binary installation has also been extended to users of the [Red Hat Container Development Kit](http://developers.redhat.com/products/cdk/overview). Information about client binary installation can be found in the previous release announcement ["Client Binary Installation Now Included in the ADB"](../../../../blog/2016/07/vagrant-service-manager-install-cli).
+
+The main new features of this release are:
+
+* Configuration information for Kubernetes provided as part of the `env` command
+* Client binary installation support for OpenShift, Kubernetes and docker in the Red Hat Container Development Kit
+* Client binary installation support for Kubernetes added to the ADB
+* Auto-detection of a previously downloaded `oc` executable binary on Windows
+* Unit and acceptance tests for the Kubernetes service
+* Option to enable kubernetes from a Vagrantfile as follows:
+
+```
+  config.servicemanager.services = 'kubernetes'
+```
+
+## Install Kubernetes Client Binary
+
+The command below will install the kubernetes binary, `kubectl`:
```

Comment:
* we try not to use above/below, maybe rephrase to something like "Run the following command to install the kubectl binary"
* kubernetes > Kubernetes